SparkPost/2web2ui
# FE-705 - Changed snapshots to explicit assertions

## What Changed
 - Changed snapshot tests into explicit assertions. 

### How To Test
 - Install newest version of matchbox, run the tests for `FilterSortCollection.test.js`. Should all pass
